third_party/config/monitoring/metrics/prometheus/kubernetes/kube-state-metrics.yaml

```diff
@@ -39,7 +39,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: kube-state-metrics
-  namespace: knative-monitoring
```

Comment:
Is it possible to keep this `ClusterRole` and `ClusterRoleBinding` in `knative-monitoring` namespace?

Reply:
I think ClusterRole and ClusterRoleBinging are k8s cluster resources. They are not belong to a specific namespace. 
But it is OK, if set namespace for them. It has not any problems.

Reply:
Sorry, just took another look. Other ClusterRole and ClusterRoleBinging are in the default namespace. I think you're right and we should keep consistence.